sw/device/lib/testing/flash_ctrl_testutils.c

```diff
@@ -358,3 +360,13 @@ status_t flash_ctrl_testutils_show_faults(
 
   return OK_STATUS();
 }
+
+status_t flash_ctrl_testutils_is_testrom(bool *is_testrom) {
+  *is_testrom =
+      retention_sram_get()
+          ->creator
+          .reserved[ARRAYSIZE((retention_sram_t){0}.creator.reserved) - 1] ==
+      TEST_ROM_IDENTIFIER;
+
+  return OK_STATUS();
+}
```

Comment:
There's no flash_ctrl in this function. Move to ret_sram_testutils.c instead?

Reply:
Heh, I was a little late with that comment, but underlying it was the test failure analysis: flash_ctrl_testutils is part of the test ROM, but englishbreakfast doesn't have the retention SRAM, so its test ROM build broke.